neo4j/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# integration pip requirements
+time
+requests
```

Comment:
We should be able to skip requests and simplejson as they are already included in the core agent.  It would be better to avoid including them again here so that we aren't requiring differing versions that may conflict.

https://github.com/DataDog/dd-agent/blob/master/requirements.txt#L27